boa/src/property/mod.rs

```diff
@@ -118,11 +118,11 @@ impl From<DataDescriptor> for PropertyDescriptor {
 #[derive(Debug, Clone, Trace, Finalize)]
 pub struct AccessorDescriptor {
     /// The function serving as getter.
-    get: Option<GcObject>,
+    pub get: Option<GcObject>,
```

Comment:
Why are the fields public? we have accessors for them.

Reply:
I needed to put them public to use pattern matching in `GcObject::get/set`, but I can rewrite it without pattern matching.

Reply:
> I needed to put them public to use pattern matching in `GcObject::get/set`, but I can rewrite it without pattern matching.

In that case make it `pub(crate)` so it's only public to the crate level.